tests/integration/test_sqlalchemy_integration.py

```diff
@@ -20,8 +23,16 @@
 @pytest.fixture
 def trino_connection(run_trino, request):
     _, host, port = run_trino
-    engine = sqla.create_engine(f"trino://test@{host}:{port}/{request.param}",
-                                connect_args={"source": "test", "max_attempts": 1})
+
+    engine = sqla.create_engine(
```

Comment:
I needed to explicitly set the schema as part of the engine otherwise Trino barfed when trying to create the views with the `Schema must be specified when session schema is not set` error.